tests/client/test_init.py

```diff
@@ -5,7 +5,7 @@ def test_resource_leaking_with_several_inits(mocked_client):
     dataset = "test_resource_leaking_with_several_inits"
     api.delete(dataset)
 
-    for i in range(0, 1000):
+    for i in range(0, 20):
```

Comment:
```suggestion
    # TODO: review performance in Windows. See https://github.com/recognai/rubrix/pull/1702
    for i in range(0, 20):
```

Just to keep in mind the performance limits.